src/pages/DelegatoryValidator/Components/MyDepositsStatusTooltip.tsx

```diff
@@ -0,0 +1,106 @@
+import {
+  Chip,
+  Stack,
+  Step,
+  StepContent,
+  StepLabel,
+  Stepper,
+  useTheme,
+} from "@mui/material";
+import React from "react";
+import TableTooltip from "../../../components/Table/TableTooltip";
+import LockIcon from "@mui/icons-material/Lock";
+import TooltipTypography from "../../../components/TooltipTypography";
+import PendingIcon from "@mui/icons-material/Pending";
+import CheckCircleIcon from "@mui/icons-material/CheckCircle";
+import {
+  STAKED_BACKGROUND_COLOR_DARK,
+  STAKED_BACKGROUND_COLOR_LIGHT,
+  STAKED_DESCRIPTION,
+  STAKED_LABEL,
+  STAKED_TEXT_COLOR_DARK,
+  STAKED_TEXT_COLOR_LIGHT,
+  WITHDRAW_PENDING_BACKGROUND_COLOR_DARK,
+  WITHDRAW_PENDING_BACKGROUND_COLOR_LIGHT,
+  WITHDRAW_PENDING_DESCRIPTION,
+  WITHDRAW_PENDING_LABEL,
+  WITHDRAW_PENDING_TEXT_COLOR_DARK,
+  WITHDRAW_PENDING_TEXT_COLOR_LIGHT,
+  WITHDRAW_READY_BACKGROUND_COLOR_DARK,
+  WITHDRAW_READY_BACKGROUND_COLOR_LIGHT,
+  WITHDRAW_READY_DESCRIPTION,
+  WITHDRAW_READY_LABEL,
+  WITHDRAW_READY_TEXT_COLOR_DARK,
+  WITHDRAW_READY_TEXT_COLOR_LIGHT,
+} from "../constants";
+
+export default function MyDepositsStatusTooltip() {
+  const theme = useTheme();
+  const steps = [
+    {
+      label: STAKED_LABEL,
+      description: STAKED_DESCRIPTION,
+      icon: <LockIcon />,
+      sxLight: {
+        color: STAKED_TEXT_COLOR_LIGHT,
+        backgroundColor: STAKED_BACKGROUND_COLOR_LIGHT,
+      },
+      sxDark: {
+        color: STAKED_TEXT_COLOR_DARK,
+        backgroundColor: STAKED_BACKGROUND_COLOR_DARK,
+      },
+    },
+    {
+      label: WITHDRAW_PENDING_LABEL,
+      description: WITHDRAW_PENDING_DESCRIPTION,
+      icon: <PendingIcon />,
+      sxLight: {
+        color: WITHDRAW_PENDING_TEXT_COLOR_LIGHT,
+        backgroundColor: WITHDRAW_PENDING_BACKGROUND_COLOR_LIGHT,
+      },
+      sxDark: {
+        color: WITHDRAW_PENDING_TEXT_COLOR_DARK,
+        backgroundColor: WITHDRAW_PENDING_BACKGROUND_COLOR_DARK,
+      },
+    },
+    {
+      label: WITHDRAW_READY_LABEL,
+      description: WITHDRAW_READY_DESCRIPTION,
+      icon: <CheckCircleIcon />,
+      sxLight: {
+        color: WITHDRAW_READY_TEXT_COLOR_LIGHT,
+        backgroundColor: WITHDRAW_READY_BACKGROUND_COLOR_LIGHT,
+      },
+      sxDark: {
+        color: WITHDRAW_READY_TEXT_COLOR_DARK,
+        backgroundColor: WITHDRAW_READY_BACKGROUND_COLOR_DARK,
+      },
+    },
+  ];
+
+  return (
+    <TableTooltip title="Deposit Status">
+      <Stack>
+        <Stepper orientation="vertical">
```

Comment:
setting `activeStep={-1} ` will do the color trick

Reply:
this will do the color trick but will hide all the descriptions. if we decide to add text as description ourselves instead of  using the `StepContent`, we are not taking advantage over already existing layouts. to manage layouts (pixel wise) ourselves, its not an easy work as i tried before and failed lol

Reply:
yeah up to you how you want to handle the icon color, i'm sure there're other ways to change it

Reply:
Btw It's a good habit to add a TODO in code if we left something to be further solved.